tooling/cli/src/interface/rust.rs

```diff
@@ -334,6 +335,18 @@ fn lookup<F: FnMut(FileType, PathBuf)>(dir: &Path, mut f: F) {
   }
 }
 
+// Copied from https://github.com/rust-lang/cargo/blob/69255bb10de7f74511b5cef900a9d102247b6029/src/cargo/core/workspace.rs#L665
+fn expand_member_path(path: &Path) -> crate::Result<Vec<PathBuf>> {
+  let Some(path) = path.to_str() else {
+    return Ok(Vec::new());
```

Comment:
Seems like an unnecessary allocation, either return an Option or Err here, we catch it anyways.

Reply:
man, let me lazily copy paste for once 😂

Reply:
naughty fabian 😏